### PR TITLE
Update delete command to accept a list of indices

### LINKED ISSUE #20: DeleteCommand-Edit command
Edit command such that user can delete 1 or more entries at one go.

### PR DESCRIPTION
**Pull request by simran-bhadani3**
Fix #20

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/index/Index.java [hunk 1]
```diff
@@ -51,4 +51,15 @@ public boolean equals(Object other) {
...
     }
+
+    @Override
+    public int compareTo(Index index) {
+        if (this.zeroBasedIndex > index.zeroBasedIndex) {
+            return 1;
+        } else if (this.zeroBasedIndex == index.zeroBasedIndex) {
+            return 0;
+        } else {
+            return -1;
```
Comment (RuiXiong2211) [inline src/main/java/seedu/address/commons/core/index/Index.java]: This can be shortened to just this.zeroBasedIndex - index.zeroBasedIndex. (I think)

**Comment by RuiXiong2211** [comment]
LGTM

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -24,30 +26,47 @@
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
 
-    private final Index targetIndex;
+    public static final String MESSAGE_DELETE_PEOPLE_SUCCESS = "Deleted People: %1$s";
```
Comment (xiinweii98) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java]: Should be "order"

Comment (xiinweii98) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java]: Same for other usages of "people" and/or "person" for this file

**Review by xiinweii98 — COMMENTED** [review]
Like you mentioned, functionality of IndexList isn't really tested as it is. Other than that, looks good.

**Pull request merged**